app/controllers/users/emails_controller.rb

```diff
@@ -62,6 +63,18 @@ def delete
       redirect_to account_url
     end
 
+    def properties
```

Comment:
`properties` is a pretty generic method name for a value that's rather specific to being parsed referer details.

Maybe something like...

```suggestion
    def parsed_referer
```